.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,5 +3,6 @@
 - [ ] Code reviewed
 - [ ] Code unit/integration/system tested
 - [ ] User documentation updated
+- [ ] (If you have changed REST API interface) view-swagger.ftl updated
```

Comment:
@dennishendriksen Could merge this with the line above: [ ] User and swagger documentation updated